composer.json

```diff
@@ -33,13 +33,14 @@
         "symfony/property-access": "^2.8 || ^3.2 || ^4.0",
         "symfony/security-csrf": "^2.8 || ^3.2 || ^4.0",
         "symfony/translation": "^2.8 || ^3.2 || ^4.0",
-        "symfony/twig-bridge": "^2.8 || ^3.2 || ^4.0,!= 4.4.0",
+        "symfony/twig-bridge": "^2.8 || ^3.2 || ^4.0",
         "symfony/validator": "^2.8 || ^3.2 || ^4.0",
         "twig/extensions": "^1.5",
         "twig/twig": "^1.34 || ^2.0"
     },
     "conflict": {
-        "sonata-project/exporter": "< 1.11.0"
+        "sonata-project/exporter": "< 1.11.0",
+        "symfony/translation-contracts": ">=2.0.0"
```

Comment:
Since `symfony/translation-contracts` requires PHP 7.2, this should not be necessary ?

Reply:
Oh you're correct, that's great, let me change that :)